Taarafo.Core/Brokers/Storages/StorageBroker.PostReports.cs

```diff
@@ -0,0 +1,14 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+//
+using Microsoft.EntityFrameworkCore;
+using Taarafo.Core.Models.Posts;
+
+namespace Taarafo.Core.Brokers.Storages
+{
+    public partial class StorageBroker
+    {
+		public DbSet<PostReport> PostReports { get; set; }
+	}
```

Comment:
Please fix the indentations in this file.
So the closing squiggly brackets need to be at the same level as the opening ones.
And the data set property needs to be one tab after that.